kge/model/embedder/lookup_embedder.yaml

```diff
@@ -39,9 +39,6 @@ lookup_embedder:
     # l_p norm to use. Negative numbers mean do not normalize.
     p: -1.                    # common choices: 1., 2.
 
-    # Whether to push gradients through normalization.
-    with_grad: False
-
```

Comment:
Good to remove this, but make sure the old configs still run (when this is set to false).

Reply:
So throw error when "with_grad: True" and run for "with_grad: False"?

Reply:
Yes, at the place where we process deprecated options.

Reply:
I know if you look at the changed files this partly already done in this PR.

Reply:
Yes, once you make this PR off master it will be easy to find ;)